ext/standard/tests/misc/gh14774.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+GH-14774 time_sleep_until overflow
+--SKIPIF--
+<?php
+if (PHP_INT_SIZE != 8) die("skip this test is for 64bit platform only");
+?>
+--FILE--
+<?php
+static $var = ((PHP_INT_MAX * 2) / 1000000000) + 1;
```

Comment:
how is `PHP_INT_MAX * 2` supposed to work. That's overflow already isn't it? Why don't you just do `PHP_INT_MAX / 500000000 + 1`?